images/twitter.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Challenge</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+
+<body class="body">
+
+    <!--Left Menu Bar-->
+<div class="left-menu-bar-section">
+    <div class="left-menu-header">
+        <a class="logo"><img src="./Assets/the-avengers-svgrepo-com.svg"></a> 
+    </div>
+
+    <!--Nav Bar-->
+
+    <div class="nav">
+        <a class="home">
+            <img src="/html-twitter-challenge/images/Assets/homepage-svgrepo-com.svg">
+            <span>Home<span>
+        </a>
+
+        <a class="explore-icon">
+            <img src="/html-twitter-challenge/images/Assets/earth-grid-svgrepo-com.svg">
+            <span>Explore</span>
+        </a>
+
+        <a class="notifications-icon">
+            <img src="/html-twitter-challenge/images/Assets/smartphone-svgrepo-com.svg">
+            <span>Notifications</span>
+        </a>
+
+        <a class="messages-icon">
+            <img src="/html-twitter-challenge/images/Assets/mailbox-svgrepo-com.svg">
+            <span>Messages</span>
+        </a>
+
+        <a class="bookmarks-icon">
+            <img src="/html-twitter-challenge/images/Assets/small-bookmark-svgrepo-com.svg">
+            <span>Bookmarks</span>
+        </a>
+
+        <a class="profile-icon">
+            <img src="/html-twitter-challenge/images/Assets/laptop-svgrepo-com.svg">
+            <span>Profile</span>
+        </a>
+
+        <a class="more-icon">
+            <img src ="/html-twitter-challenge/images/Assets/more-svgrepo-com.svg">
+            <span>More</span>
+        </a>
+    </div>
+
+    <div class="left-tweet-button">
```

Comment:
You might want to try and wrap elements like this one in anchors, that way your cursor turns to a pointer and makes it possible to us tab to navigate the page.